mtags/src/main/scala-3/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -259,7 +282,7 @@ class CompletionProvider(
                     )
                     if (byParamCount != 0) byParamCount
                     else {
-                      s1.show.compareTo(s2.show)
+                      s1.detailString.compareTo(s2.detailString)
```

Comment:
We should compare the full signature, show for symbol was mostly just name, which wasn't an issue prior to 3.0.1, since before the completions with the same symbol name were grouped together.